packages/web/src/pages/App.js

```diff
@@ -437,9 +439,6 @@ class App extends Component {
   }
 
   acceptUpdateApp = () => {
-    if (this.state.showUpdateAppBanner) {
-      this.dismissUpdateAppBanner()
-    }
```

Comment:
Is this no longer necessary because the app is going to restart on update and not show the banner after?

Reply:
Basically there are two flows for upgrading desktop:
* An update to entire electron app (you need a new chromium basically)
* An update to just the static JS for the SPA
KJ added the latter recently, which lets us basically not worry about showing a banner for the former and letting auto update on quit take care of that behind the scenes. There's still banner that shows that will get dismissed in the below method (acceptWebUpdate) but that's for the latter SPA refresh